BootloaderCorePkg/Include/Library/BootloaderCoreLib.h

```diff
@@ -120,24 +120,80 @@ typedef struct {
   UINT8         BootPartition;
 } S3_DATA;
 
+//
+// Format to share info between bootloader and UEFI payload.
```

Comment:
Let me do not mention UEFI payload here.  The code should be applicable for any payload that will do SMM base initialization.

Reply:
Corrected.